site/pt/beta/tutorials/keras/basic_text_classification.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<p>Instead of "aproximação", I would translate "approach" into "abordagem". There are also a few typos:</p><blockquote><strong>Esta</strong> é uma <strong>abordagem</strong> ingênua que conseguiu uma acurácia de 87%. <strong>Com abordagens mais avançadas</strong>, o modelo deve chegar em 95%.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>